src/Microsoft.ComponentDetection.Contracts/Mappers/CycloneDx.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using CycloneDX.Json;
+using CycloneDX.Models;
+using Microsoft.ComponentDetection.Contracts.BcdeModels;
+
+namespace Microsoft.ComponentDetection.Contracts.Mappers
+{
+    public static class CycloneDx
+    {
+        public static string ToCycloneDx(this ScanResult scanResult)
+        {
+            return Serializer.Serialize(
+                new Bom
+                {
+                    Metadata = new Metadata
+                    {
+                        Tools = new List<Tool>
+                        {
+                            new Tool
+                            {
+                                Vendor = "Microsoft",
+                                Name = "Component Detection",
+                            },
+                        },
+                    },
+                    Components = scanResult.ComponentsFound.ToComponents(),
+                });
+        }
+
+        private static Component ToComponent(this ScannedComponent scannedComponent)
+        {
+            return new Component
+            {
+                Type = Component.Classification.Library,
+                Name = scannedComponent.Component.PackageUrl.Name,
+                Version = scannedComponent.Component.PackageUrl.Version,
+                Purl = scannedComponent.Component.PackageUrl.ToString(),
+                Properties = scannedComponent.GenerateProperties(),
+            };
```

Comment:
I'm not super familiar with this codebase. But looks to me like once you get to a ScannedComponent there is a bit of information loss. i.e. I think I saw authors on NugetComponent and there is a hash available on GoComponent. Is it possible to include this additional information?